lib/shipit/command.rb

```diff
@@ -58,7 +58,7 @@ def success?
     end
 
     def exit_message
-      "#{self} exited with status #{@code}"
+      "#{self} #{termination_status}"
```

Comment:
Is `self` end up calling `to_s`. Should we call `to_s` directly here?

Reply:
rubocop disagrees 👮‍♀

```
lib/shipit/command.rb:62:10: W: Lint/StringConversionInInterpolation: Use self instead of Object#to_s in interpolation.
      "#{to_s} #{termination_status}"
```